stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/MessagesGapInfo.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.offline.plugin.logic.channel.internal
+
+public data class MessagesGapInfo(
+    val messagesAboveGap: List<Long>,
+    val messagesBellowGap: List<Long>
+)
```

Comment:
What do those parameters mean in the historical axis?
Does `Above` imply either `older` or `newer` messages (same for `Below`)?

Reply:
above is older and bellow is newer. The problem is that older messages than the gap may be added when loading newer messages and newer messages may be added when loading older messages... At first I thought about naming then older and newer messages than gap, but in the end I thought is was more confusing than clarifying.

Reply:
I mean the names do give me no clue if the Above is for newer or not.
It's more like a description in UI positioning like Above something.

Reply:
Let's imagine we (or customers) changed the orientation of the history axis. 
And now `Above` implies newer, while `Below` - older.

Reply:
Let's use `Forward` and `Backward` then, it still uses direction.